compiler/z/codegen/S390GenerateInstructions.cpp

```diff
@@ -1011,9 +1011,34 @@ generateRILInstruction(TR::CodeGenerator * cg, TR::InstOpCode::Mnemonic op, TR::
    return new (INSN_HEAP) TR::S390RILInstruction(op, n, treg, label, cg);
    }
 
-TR::Instruction *
-generateRILInstruction(TR::CodeGenerator * cg, TR::InstOpCode::Mnemonic op, TR::Node * n, TR::Register * treg, uintptrj_t imm, TR::Instruction * preced)
+TR::Instruction * // this overload is only for immediate values which are used directly in the instruction and not part of an address calculation
```

Comment:
These types of comments should go in the header file in a Doxygen format. Let's take the time to properly Doxygen the newly added and modified APIs and remove these single line comments in the implementation (cpp) files.